Aflores plymouth (#234)
osisoft/Edge-Data-Store-Docs: content/sds/types/sds-type-reusability.md

# SdsType reusability

> An SdsType can refer other SdsTypes by using their identifiers. This enables type reusability. For example, if there is a common index and value property for a group of types that may have additional properties, you can create a base type with those properties. This is shown in the following example.

Reviewer: ```suggestion
An `SdsType` can refer other `SdsTypes` by using their identifiers. This enables type reusability. For example, if there is a common index and value property for a group of types that may have additional properties, you can create a base type with those properties. This is shown in the following example.
```

}
```

> If a new type should be created with additional properties to the ones above, add a reference to the base type by specifying the base type's `Id`, as shown in the following example.

Reviewer: ```suggestion
If you need a new type with additional properties to the ones above, add a reference to the base type by specifying the base type's `Id`, as shown in the following example.
```
